components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/AbstractAuthorizationGrantHandlerTest.java

```diff
@@ -130,9 +151,30 @@
                 {true, false, true, 0L, 0L, 3600L, 0L, true, TOKEN_STATE_REVOKED, true},
                 {true, false, true, 0L, 0L, 0L, 0L, true, TOKEN_STATE_ACTIVE, true},
                 {true, false, true, 0L, 0L, 0L, 3600L, true, TOKEN_STATE_ACTIVE, true},
+
+                {true, false, true, 0L, 0L, -1L, 3600L, true, TOKEN_STATE_ACTIVE, true},
```

Comment:
You can use OAuthConstants.TokenStates.ACTIVE instead TOKEN_STATE_ACTIVE. This applies for all other cases